python/grass/jupyter/timeseriesmap.py

```diff
@@ -477,24 +384,20 @@ def save(
         param str text_color: color to use for the text.
         """
 
-        # Render images if they have not been already
-        if not self._layers_rendered:
-            self.render()
-
         input_files = []
         for date in self._dates:
             input_files.append(self._date_filename_dict[date])
 
-        save_gif(
-            input_files,
-            filename,
+        return super().save(
+            filename=filename,
+            save_files=input_files,
+            labels=self._dates,
+            labels=self._names,
```

Comment:
This results in `SyntaxError: keyword argument repeated`, breaking compilation.

Reply:
This worked. Thank you. But I still need clarification about what's wrong while running the tests locally; they all get passed despite such errors.

Reply:
Are you rebuilding the code after changes? You need to run `make` in the jupyter folder.

Reply:
No, I was not rebuilding the code earlier. I tried rebuilding it this time by doing `reload()`. From my side, the notebook ran without any errors.

Reply:
Just to confirm, were you able to rebuild the code with make?

SeriesMap is currently not working, in `widgets.SelectionSlider()` parameter description can't be None but it apparently needs to be empty string.